app/controllers/spree/admin/orders/customer_details_controller_decorator.rb

```diff
@@ -0,0 +1,33 @@
+module Spree
+  module Admin
+    module Orders
+      module CustomerDetailsControllerDecorator
+        def self.prepended(base)
+
+        end
+
+
+        def update
+          params[:order][:user_id] = nil if guest_checkout?
+          if @order.update(order_params)
+            @order.update_with_updater! # Causes Avatax to recalculate tax and totals
```

Comment:
Just confirming this is the only line added, otherwise it's the same? @dudemanvox